operating/examine-images-for-vulnerabilities.adoc

```diff
@@ -7,16 +7,28 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-With {product-title} you can analyze images for vulnerabilities.
-Scanner analyzes all image layers to check for known vulnerabilities by comparing them with the Common Vulnerabilities and Exposures (CVEs) list.
+With {product-title} you can analyze images for vulnerabilities using the default {product-title-short} scanner, or you can xref:../integration/integrate-with-image-vulnerability-scanners.adoc#integrate-with-image-vulnerability-scanners[configure an integration] to use another supported scanner. The  scanners in {product-title-short} analyze all image layers to check for known Common Vulnerabilities and Exposures (CVEs) by comparing them with a vulnerability database populated from different sources. Depending on the scanner used, sources include the National Vulnerability Database (NVD), the Open Source Vulnerabilities (OSV) vulnerability database, and operating system OVAL feeds.
```

Comment:
The wording isn't quite right here:

>  The  scanners in {product-title-short} analyze all image layers to check for known Common Vulnerabilities and Exposures (CVEs) by comparing them with a vulnerability database populated from different sources.

They analyze each image layer to find packages which we then match against known vulnerabilities (not necessarily CVEs)

> Depending on the scanner used, sources include the National Vulnerability Database (NVD), the Open Source Vulnerabilities (OSV) vulnerability database, and operating system OVAL feeds.

Replace OVAL with vulnerability